### DISCUSSION TITLE
createToken usage

### DISCUSSION THREAD

**Discussion by jun-gh** [flutter-stripe/flutter_stripe#1054] [Q&A]
How to use the createToken?
I checked it needs CreateTokenParams, how do I get this data from the stripe card?

**Comment by jamesdixon** [comment]
@jun-gh did you ever figure this out? It's not clear to me at all...

**Reply by jun-gh** [reply]
@jamesdixon Yes. 
On client side, I run the createToken to get token.id. Then pass the token.id to backend. 
On backend side, I run the stripe->create->charge api, passing the token.id as source.
